src/Microsoft.Health.Fhir.Liquid.Converter/Processors/CcdaProcessor.cs

```diff
@@ -4,26 +4,34 @@
 // -------------------------------------------------------------------------------------------------
 
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using DotLiquid;
 using Microsoft.Health.Fhir.Liquid.Converter.Models;
 using Microsoft.Health.Fhir.Liquid.Converter.Parsers;
+using Microsoft.Health.Fhir.Liquid.Converter.Telemetry;
+using Microsoft.Health.Logging.Telemetry;
 
 namespace Microsoft.Health.Fhir.Liquid.Converter.Processors
 {
     public class CcdaProcessor : BaseProcessor
     {
         private readonly IDataParser _parser = new CcdaDataParser();
 
-        public CcdaProcessor(ProcessorSettings processorSettings)
-            : base(processorSettings)
+        public CcdaProcessor(ProcessorSettings processorSettings, ITelemetryLogger telemetryLogger)
+            : base(processorSettings, telemetryLogger)
         {
         }
 
-        public override string Convert(string data, string rootTemplate, ITemplateProvider templateProvider, TraceInfo traceInfo = null)
+        protected override string InternalConvert(string data, string rootTemplate, ITemplateProvider templateProvider, TraceInfo traceInfo = null)
         {
-            var ccdaData = _parser.Parse(data);
-            return Convert(ccdaData, rootTemplate, templateProvider, traceInfo);
+            object ccdaData;
+            using (ITimed templateRetrivalTime = TelemetryLogger.TrackDuration(ConverterMetrics.InputDeserializationDuration))
```

Comment:
nit: rename `templateRetrivalTime` to reflect the metric InputDeserializationDuration. 
Same for below classes.

Reply:
Right, thanks for pointing out.